test/Components.re

```diff
@@ -242,3 +242,12 @@ module ShouldAllowComponentProp = {
     hooks,
   );
 };
+
+module LocallyAbstractType : {
+  let make: (~key: Key.t=?, ~foo: int, ~onWhatever: int => unit, unit) => element(node);
+} = {
+  let%component make = (type a, ~foo: a, ~onWhatever: a => unit, (), hooks) => {
```

Comment:
Thanks for that. It might be easy to accidentally remove this text. I don't want to over engineer the solution but a comment would be great.

Reply:
Done.